runelite-client/src/main/java/net/runelite/client/plugins/menuentryswapper/MenuEntrySwapperPlugin.java

```diff
@@ -1063,14 +1063,19 @@ else if (target.contains("explorer's ring") && this.getSwapExplorersRing)
 					break;
 			}
 		}
-
-		if (option.equals("talk-to"))
+		
+		if (option.equals("talk-to") || option.equals("attack"))
```

Comment:
I feel this will create unintended issues, they should remove attack option if the intent is to pickpocket.

Reply:
ive been testing it today in PVP worlds, it works fine - monsters with attack and no pickpocket still show attack, players still show attack too